src/GitVersionCore/Extensions/GitExtensions.cs

```diff
@@ -9,19 +9,19 @@ namespace GitVersion.Extensions
 {
     public static class GitExtensions
     {
-        public static DateTimeOffset When(this Commit commit)
+        public static DateTimeOffset When(this ICommit commit)
         {
             return commit.CommitterWhen.Value;
         }
 
-        public static string NameWithoutRemote(this Branch branch)
+        public static string NameWithoutRemote(this IBranch branch)
```

Comment:
Any reason not to provide `NameWithoutRemote` as a property on the `IBranch` interface?

Reply:
Agree, I will change that in a subsequent PR